From 72a11f11d8f54ecd0bf02fb5419f1d9a2557cb39 Mon Sep 17 00:00:00 2001
From: Xiao Ya peng <yp.xiao@rock-chips.com>
Date: Tue, 26 Nov 2019 09:09:21 +0800
Subject: [PATCH] ARM: dts: rockchip: rk322x: Add the pwm2/pwm3 pinctrl with
 pull up

The default state of pwm2/pwm3 pin is a pull up, we need to keep it
for the second global reset issue, when pwm2/pwm3 used for the regulator

Change-Id: Ic57e26338f7f546e3f784bd34790ed9b027d78de
Signed-off-by: Xiao Ya peng <yp.xiao@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index 463017ce1bc3..5a2cbad76244 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -1588,6 +1588,10 @@
 				rockchip,pins = <1 12 RK_FUNC_2 &pcfg_pull_none>;
 			};
 
+			pwm2_pin_pull_up: pwm2-pin-pull-up {
+				rockchip,pins = <1 12 RK_FUNC_2 &pcfg_pull_up>;
+			};
+
 			pwm2_pin_pull_down: pwm2-pin-pull-down {
 				rockchip,pins = <1 12 RK_FUNC_2 &pcfg_pull_down>;
 			};
@@ -1598,6 +1602,10 @@
 				rockchip,pins = <1 11 RK_FUNC_2 &pcfg_pull_none>;
 			};
 
+			pwm3_pin_pull_up: pwm3-pin-pull-up {
+				rockchip,pins = <1 11 RK_FUNC_2 &pcfg_pull_up>;
+			};
+
 			pwm3_pin_pull_down: pwm3-pin-pull-down {
 				rockchip,pins = <1 11 RK_FUNC_2 &pcfg_pull_down>;
 			};
-- 
2.35.3

